cli/internal/helm/loader_test.go

```diff
@@ -20,27 +20,56 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/chartutil"
 	"helm.sh/helm/v3/pkg/engine"
 
+	"github.com/edgelesssys/constellation/v2/cli/internal/clusterid"
+	"github.com/edgelesssys/constellation/v2/cli/internal/terraform"
 	"github.com/edgelesssys/constellation/v2/internal/attestation/idkeydigest"
 	"github.com/edgelesssys/constellation/v2/internal/attestation/measurements"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/azureshared"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/gcpshared"
 	"github.com/edgelesssys/constellation/v2/internal/config"
 )
 
+func fakeServiceAccURI(provider cloudprovider.Provider) string {
+	switch provider {
+	case cloudprovider.GCP:
+		cred := gcpshared.ServiceAccountKey{
+			Type:                    "service_account",
+			ProjectID:               "project_id",
+			PrivateKeyID:            "key_id",
+			PrivateKey:              "key",
+			ClientEmail:             "client_email",
+			ClientID:                "client_id",
+			AuthURI:                 "auth_uri",
+			TokenURI:                "token_uri",
+			AuthProviderX509CertURL: "cert",
+			ClientX509CertURL:       "client_cert",
+		}
+		return cred.ToCloudServiceAccountURI()
+	case cloudprovider.Azure:
+		creds := azureshared.ApplicationCredentials{
+			TenantID:            "TenantID",
+			Location:            "Location",
+			PreferredAuthMethod: azureshared.AuthMethodUserAssignedIdentity,
+			UamiResourceID:      "uid",
+		}
+		return creds.ToCloudServiceAccountURI()
+	default:
+		return ""
+	}
+}
```

Comment:
If we moved marshalling up into `init.go`, this can be replaced by arbitrary `[]byte` values. Highlighting the fact that this package does not interact with the values it receives.